### PR TITLE
moved files into packages and formatter plugin changed

### PR DESCRIPTION
**Pull request by aslanbekirov**
moved files into packages and maven java formatter plugin configuration file tag added.

### CODE REVIEW

**Comment by elingg** [comment]
LGTM. Quick question, why are you removing the @Override tag? If we aren't implementing those methods, can we safely remove them?

**Comment by elingg** [comment]
Ok, good point. Could you rebase with master and update the POM to point to 1.6 instead of 1.5 in this case? Then we should be good to merge.

**Comment by aslanbekirov** [comment]
@elingg , maven compiler plugin added and point to 1.6

File: java/pom.xml [hunk 1]
```diff
@@ -1,52 +1,64 @@
...
-            <name>Mesosphere Repo</name>
-            <url>http://downloads.mesosphere.io/maven</url>
-        </repository>
-    </repositories>
-    <dependencies>
-        <dependency>
-            <groupId>org.apache.mesos</groupId>
-            <artifactId>mesos</artifactId>
-            <version>0.21.1</version>
-        </dependency>
```
Comment (elingg) [inline java/pom.xml]: Could we change only the version number in the pom.xml instead of reformatting?

**Comment by elingg** [comment]
Could you fix the formatting in pom.xml and also rebase with master? Thanks!

**Comment by aslanbekirov** [comment]
Hi @elingg , I rebased with master and pom.xml formatting fixed. Thanks.

**Comment by elingg** [comment]
:+1:

**Comment by aslanbekirov** [comment]
:+1:  :) , Thanks, @elingg

**Pull request merged**